src/main/java/uk/gov/hmcts/reform/idam/web/helper/AuthHelper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.idam.web.helper;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class AuthHelper {
+
+    private final boolean useSecureCookie;
+
+    public AuthHelper(@Value("${authentication.secureCookie}") boolean useSecureCookie) {
+        this.useSecureCookie = useSecureCookie;
+    }
+
+    public List<String> makeCookiesSecure(List<String> cookies) {
+        return cookies.stream()
+            .map(cookie -> {
+                if (!cookie.contains("HttpOnly")) {
```

Comment:
What if the cookie already has `HttpOnly` but not `Secure`?

Reply:
@radoslaw-orlowski-amido yeah i see what your saying. @nikola-naydenov-hmcts i think created this method originally, do you think it is worth putting in any further checks for the `Secure` like the `HttpOnly` flag? I'm not sure if there will ever be case a cookie with the `Secure` flag being passed in.

Reply:
One note, I am aware that probably this situation will never occur in the current code state. I'm just trying to make sure it works correctly in the future, in case we introduce such a problematic cookie (for whatever reason).

Reply:
good point by @radoslaw-orlowski-amido here

they should always be HttpOnly
and they should be Secure if the feature flag is set ( we disable in local servers because we don't use https )

Reply:
@radoslaw-orlowski-amido @tbamido broke up the if statements